src/enums.h

```diff
@@ -15,6 +15,25 @@
 #ifndef SRC_ENUMS_H_
 #define SRC_ENUMS_H_
 
+#include <algorithm>
+#include <string>
+
 typedef enum { UNKNOWN = -1, JPEG2000 = 0, JPEG = 1, RAW = 2 } DCM_Compression;
 
+inline DCM_Compression dcmCompressionFromString(std::string compressionStr) {
+  DCM_Compression compression = UNKNOWN;
+  std::transform(compressionStr.begin(), compressionStr.end(),
+                 compressionStr.begin(), ::tolower);
+  if (compressionStr.find("jpeg") == 0) {
```

Comment:
Won't this trigger for "jpeg2000"? Why use find() instead of equals()? Also can we add a unit test for this method?

Reply:
> Why use find() instead of equals()
do you meant compare()? to cover additional chars what may be entered by user
> Won't this trigger for "jpeg2000"?
yes, but would be overridden, since it called one time per file i didn't put jpeg200 if into jpeg if

Reply:
What is someone puts in a variant of JPEG we don't support? E.g. jpegls or some other obscure one. They might think they're getting lossless compression, but we'll just be using lossy jpeg.

I think we should be more strict here.

Reply:
done